ocaml/xapi/xapi_pool_update.ml

```diff
@@ -259,6 +260,13 @@ let parse_update_info xml =
       with
       | _ -> []
     in
+    let enforce_homogeneity =
+      try
+        List.assoc "enforce-homogeneity" attr
+        |> fun s -> s = "1" || String.lowercase s = "true"
```

Comment:
I think to remember an helper somewhere in xapi that does exactly this. It would be better to use the helper to reduce the changes when updating compiler: `String.lowercase` has been deprecated and renamed `String.lowercase_ascii` in OCaml 4.03.0